src/common/query/src/logical_plan.rs

```diff
@@ -79,6 +82,71 @@ pub fn rename_logical_plan_columns(
     LogicalPlanBuilder::from(plan).project(projection)?.build()
 }
 
+/// Convert a insert into logical plan to an (table_name, logical_plan)
+/// where table_name is the name of the table to insert into.
+/// logical_plan is the plan to be executed.
+///
+/// if input logical plan is not `insert into table_name <input>`, return None
+pub fn breakup_insert_plan(
+    plan: &LogicalPlan,
+    catalog: &str,
+    schema: &str,
```

Comment:
these two may not be used if there are catalog and schema in the input plan (which is likely to have). I'd suggest to add some comments about this behavior

Reply:
> these two may not be used if there are catalog and schema in the input plan (which is likely to have). I'd suggest to add some comments about this behavior

substrait `WriteRel` require not-optional catalog&schema for table name

Reply:
I mean the in-param may not be used in the plan, if for example the plan is `TableReference::Full`